00_hello_world/main.go

```diff
@@ -8,6 +8,19 @@ import (
 
 var out io.Writer = os.Stdout
 
+func fib(i int) int {
+	a, b := 0, 1
+	fmt.Fprintln(out, b)
```

Comment:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37